pkg/deploy/deployment_che.go

```diff
@@ -179,6 +180,24 @@ func NewCheDeployment(cr *orgv1.CheCluster, cheImageAndTag string, cmRevision st
 		},
 	}
 
+	cheMultiUser := GetCheMultiUser(cr)
+	if cheMultiUser == "false" {
+		cheDeployment.Spec.Template.Spec.Volumes = []corev1.Volume{
+			{
+				Name: DefaultCheVolumeName,
+				VolumeSource: corev1.VolumeSource{
+					PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+						ClaimName: DefaultCheVolumeName,
```

Comment:
I think it should be called DefaultCheVolumeClaimName, because we don't create volume itself. che-operator creates volumeClaim and infra provide real volume for us.

Reply:
Ok I see, we are using the same name for volume and claim, so let it be.